orderbook/openapi.yml

```diff
@@ -471,6 +502,16 @@ components:
           description: "Hash of the corresponding settlement transaction containing the trade (if available)."
           $ref: "#/components/schemas/TransactionHash"
           nullable: true
+      required:
+        - blockNumber
+        - logIndex
+        - orderUid
+        - owner
+        - sellToken
+        - buyToken
+        - sellAmount
+        - sellAmountBeforeFees
+        - buyAmount
```

Comment:
If there is no transaction hash do we return null or do we omit to serialized the field? In the former case the field would still be required, but nullable.

Reply:
Interesting fact. We return null. Will include this then. Are there any other fields that fit this description I am missing?